e2e/cypress/tests/integration/system_console/true_up_review_spec.js

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import moment from 'moment-timezone';
+
+describe('System Console > Site Statistics > True Up Review', () => {
+    before(() => {
+        cy.apiInitSetup();
+        cy.apiRequireLicense();
+    });
+
+    it('cloud instances should not see true up review sections', () => {
+        cy.apiRequireLicense('cloud');
+
+        // # Go to team statistics
+        cy.visit('/admin_console/reporting/team_statistics');
+        cy.findByText('True Up Review').should('not.exist');
+
+        // # Go to site statistics
+        cy.visit('/admin_console/reporting/system_analytics');
+        cy.findByText('True Up Review').should('not.exist');
+    });
```

Comment:
Should this be logged in as admin to make sure they don't show due to cloud license and not because they are not an admin user?

Reply:
Ah, good point! Definitely, I'll update the tests.